index.js

```diff
@@ -31,17 +31,13 @@ const POLLING_INTERVAL = 0.2;
 const POLLING_MAX_ATTEMPT = 10;
 const STATE_MAP = {
     SUCCESS: 'success',
-    RUNNING: 'pending',
-    QUEUED: 'pending',
-    PENDING: 'pending'
+    PENDING: 'pending',
+    FAILURE: 'failure'
 };
 const DESCRIPTION_MAP = {
     SUCCESS: 'Everything looks good!',
     FAILURE: 'Did not work as expected.',
-    ABORTED: 'Aborted mid-flight',
-    RUNNING: 'Testing your code...',
-    QUEUED: 'Looking for a place to park...',
-    PENDING: 'Looks good but incomplete.'
```

Comment:
are these statuses no longer valid?

Reply:
they should map to statuses from https://docs.github.com/en/rest/reference/repos#statuses

input to scm-* functions from models should be mapped to actual scm status supported by API

Reply:
They are no longer valid.